internal/manager/ecs/native.go

```diff
@@ -90,6 +92,13 @@ func (e *Manager) deployLocal(w io.Writer) error {
 			pterm.Println("Failed to get logs:", err)
 		}
 
+		sr, err := getStoppedReason(e.App.Cluster, name, svc)
+		if err != nil {
+			return err
+		}
+
+		pterm.Printfln("Stopped reason: %s", sr)
```

Comment:
Let's add a bit more info on the error message - I'd like to show the container name:
```suggestion
		pterm.Printfln("Container %s couldn't start: %s", name, sr)
```